qa/suites/fs/verify/validater/valgrind.yaml

```diff
@@ -1,5 +1,5 @@
-# see http://tracker.ceph.com/issues/20360 and http://tracker.ceph.com/issues/18126
-os_type: centos
+# Only works on os_type: centos
+# See http://tracker.ceph.com/issues/20360 and http://tracker.ceph.com/issues/18126
```

Comment:
@batrick , I see all fs-suite valgrind tests (4 jobs) in ubuntu failing due to an another issue. 
http://pulpito.ceph.com/?branch=wip-yuri3-testing-2020-05-27-1545-nautilus
e.g., http://pulpito.ceph.com/yuriw-2020-05-27_18:23:59-fs-wip-yuri3-testing-2020-05-27-1545-nautilus-distro-basic-smithi/5096036/
Is it OK if we still merge this?

Reply:
Needs: https://tracker.ceph.com/issues/45804

@smithfarm  can you add that one into this PR too.